projects/muparser/set_eval_fuzzer.cc

```diff
@@ -0,0 +1,32 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <stdint.h>
+#include <stdio.h>
+#include <iostream>
+#include <string>
+
+#include "muParser.h"
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
+  std::string line_string((char *)data, size);
+  try {
+    mu::Parser parser;
+    parser.SetExpr(line_string);
+    parser.Eval();
+  } catch (mu::Parser::exception_type &e) {
+     // std::cout << e.GetMsg() << std::endl;
```

Comment:
By comment out, i mean use a comment like
// Skip logging errors to avoid log spam.

Reply:
I see. Thanks.